### PR TITLE
Update Content from byte array to base64string

### PR DESCRIPTION
**Pull request by KostinP**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/264?issue_count=23&issue_position=4&next_issue_id=255&prev_issue_id=265

### CODE REVIEW

**Review by borovHokage — APPROVED** [review]

File: test/FileService.Broker.UnitTests/GetFileConsumerTests.cs [hunk 1]
```diff
@@ -38,8 +38,7 @@ public void SetUp()
                 new GetFileConsumer(repository.Object));
 
             fileId = Guid.NewGuid();
-            contentByte = Convert.FromBase64String("RGlnaXRhbCBPZmA5Y2U=");
-            contentString = Convert.ToBase64String(contentByte);
+            contentString = "RGlnaXRhbCBPZmA5Y2U=";
```
Comment (djino97) [inline test/FileService.Broker.UnitTests/GetFileConsumerTests.cs:41]: necessary fields 
image

File: test/FileService.Mappers.UnitTests/ImageMapperTests.cs [hunk 2]
```diff
@@ -73,7 +73,7 @@ public class ImageMapperTests
         private FakeHttpContextAccessor fakeHttpContextAccessor;
 
         private ImageRequest imageRequest;
-        private byte[] resizedImageContent;
+        private string resizedImageContent;
```
Comment (djino97) [inline test/FileService.Mappers.UnitTests/ImageMapperTests.cs:76]: resolve warnings 
image

File: test/FileService.Mappers.UnitTests/ImageMapperTests.cs [hunk 3]
```diff
@@ -118,7 +118,7 @@ public void SetUp()
         //    var expectedImage = new DbImage
         //    {
         //        Id = newImage.Id,
-        //        Content = Convert.FromBase64String(imageRequest.Content),
+        //        Content = imageRequest.Content,
         //        Extension = imageRequest.Extension,
         //        Name = imageRequest.Name,
```
Comment (djino97) [inline test/FileService.Mappers.UnitTests/ImageMapperTests.cs:123]: is it necessary comments?

File: test/FileService.Mappers.UnitTests/ImageMapperTests.cs (hunk 2 above)
Comment (KostinP) [reply test/FileService.Mappers.UnitTests/ImageMapperTests.cs:76]: I think this is not my task

File: test/FileService.Mappers.UnitTests/ImageMapperTests.cs (hunk 3 above)
Comment (KostinP) [reply test/FileService.Mappers.UnitTests/ImageMapperTests.cs:123]: This was commented on by Spartak

File: test/FileService.Broker.UnitTests/GetFileConsumerTests.cs (hunk 1 above)
Comment (KostinP) [reply test/FileService.Broker.UnitTests/GetFileConsumerTests.cs:41]: Fixed.

**Review by MariyaSin — APPROVED** [review]

**Pull request merged**